src/main/js/screens/app/screens/quizes/QuizesScreen.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { Alert, Grid, FormControl, Button, ButtonGroup } from 'react-bootstrap';
+
+import quizService from '../../../../shared/services/quizService';
+import QuizBox from './QuizBox';
+import QuizForm from './QuizForm';
+
+class QuizesScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      quizes: [],
+      search: '',
+      offset: 0,
+      showCreate: false,
+    };
+
+    this.onInputChange = this.onInputChange.bind(this);
+    this.handleCreateQuiz = this.handleCreateQuiz.bind(this);
+    this.handleDeleteQuiz = this.handleDeleteQuiz.bind(this);
+  }
+
+  componentWillMount() {
+    this.getQuizes('', 0);
+  }
+
+  onInputChange(event) {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+    this.getQuizes(event.target.value, 0);
+  }
+

```

Comment:
Are _searchPattern_ and _offset_ needed as parameters? Aren't they accessible in the state?